### PR TITLE
Additional 42 multi gpu

### PR DESCRIPTION
**Pull request by madil90**
Fixed flake8 error and changed name to generic

### CODE REVIEW

**Comment by Nic-Ma** [comment]
Thanks for your PR.

Hi @ericspod and @wyli ,

Could you please help review it? I think it looks good to me.
Of course, glad to have a discussion if you have any comments.
Thanks.

File: monai/application/engine/multi_gpu_supervised_trainer.py [hunk 1]
```diff
@@ -50,8 +45,8 @@ def _default_eval_transform(x, y, y_pred):
 
 
 @monai.utils.export("monai.application.engine")
-def create_multigpu_supervised_trainer(net, optimizer, loss_fn, devices=None, non_blocking=False,
```
Comment (ericspod) [inline monai/application/engine/multi_gpu_supervised_trainer.py]: As I said elsewhere, I wouldn't change this name it clashes with that from Ignite. I can see people using one or the other in their code and we shouldn't introduce confusion.

Comment (Nic-Ma) [reply monai/application/engine/multi_gpu_supervised_trainer.py]: Hi @ericspod and @madil90 ,

OK, let's temporarily close and postpone this PR to when we organize all kinds of trainers.
Do you agree?
Thanks.

Comment (madil90) [reply monai/application/engine/multi_gpu_supervised_trainer.py]: Hi @Nic-Ma @ericspod I removed the original changes and added a small commit to resolve CI issues. Basically skip tests when no GPUs are available. @ericspod can you please review and merge if it looks good.

**Pull request merged**